nixos/modules/services/networking/croc.nix

```diff
@@ -0,0 +1,78 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (lib) types;
+  cfg = config.services.croc;
+in
+{
+  options.services.croc = {
+    enable = lib.mkEnableOption "croc relay";
+    ports = lib.mkOption {
```

Comment:
Generally we put an `openFirewall` option in to make sure the user really wants to open ports... but that's the entire point you're running `croc` relay :thinking:

I'm trying to think if I have seen any services which are only network oriented like this and **don't** have an `openFirewall` option.